### PR TITLE
Add gcc 4.9.4 and 5.5.0 for arm64

### PR DESCRIPTION
**Pull request by maringuu**
I hope I didn't miss something this time. gdb is disabled and fortran enabled.

### CODE REVIEW

**Comment by maringuu** [comment]
@dkm Gentle Ping :)

**Comment by dkm** [comment]
Oh, sorry about the delay. Is this linked to an existing issue, or simply an opportunistic PR to add some missing compilers? Trying to refresh my memory, sorry.

**Comment by maringuu** [comment]
This PR is the logical continuation of #65 but for arm. Iirc these two compilers are the last ones missing to support all compiler versions for arm64.

File: build/latest/arm64-4.9.4.config [hunk 1]
```diff
@@ -0,0 +1,892 @@
...
+#
+# CT_LOG_ERROR is not set
+# CT_LOG_WARN is not set
+# CT_LOG_INFO is not set
+CT_LOG_EXTRA=y
+# CT_LOG_ALL is not set
+# CT_LOG_DEBUG is not set
+CT_LOG_LEVEL_MAX="EXTRA"
+# CT_LOG_SEE_TOOLS_WARN is not set
+CT_LOG_PROGRESS_BAR=y
```
Comment (dkm) [inline build/latest/arm64-4.9.4.config:121]: You should remove this

File: build/latest/arm64-5.5.0.config [hunk 2]
```diff
@@ -0,0 +1,881 @@
...
+#
+# CT_LOG_ERROR is not set
+# CT_LOG_WARN is not set
+# CT_LOG_INFO is not set
+CT_LOG_EXTRA=y
+# CT_LOG_ALL is not set
+# CT_LOG_DEBUG is not set
+CT_LOG_LEVEL_MAX="EXTRA"
+# CT_LOG_SEE_TOOLS_WARN is not set
+CT_LOG_PROGRESS_BAR=y
```
Comment (dkm) [inline build/latest/arm64-5.5.0.config:121]: Same

**Comment by dkm** [comment]
Sorry for not getting back earlier. You should remove the progress bar as it cripples the log. I'll do it after I'm done with the gcc 14 compilers.

**Comment by maringuu** [comment]
Thanks for fixing up and your follow-up at infra!

**Pull request merged**